backend/src/database/migrations/U1702035391__track-manual-org-changes.sql

```diff
@@ -0,0 +1,2 @@
+alter table organizations
+    drop column "manuallyChanged";
```

Comment:
can we call this manuallyChangedFields or similar? looks like a boolean with this wording